src/pages/commits/commitChart/CommitChart.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { green, gray, yellow, red } = uiColors;
+
+export type TaskStats = {
+  success?: number;
+  failure?: number;
+  dispatched?: number;
+  scheduled?: number;
+  unscheduled?: number;
+  systemFailure?: number;
+  setupFailure?: number;
+  total: number;
```

Comment:
did we confirm these are the only statuses we care about? 
Also `scheduled` is not an official task status this should be `will-run`